src/k8s/pkg/k8sd/api/cluster_bootstrap.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/canonical/microcluster/state"
 )
 
-func (e *Endpoints) postClusterBootstrap(s *state.State, r *http.Request) response.Response {
+func (e *Endpoints) postClusterBootstrap(s state.State, r *http.Request) response.Response {
```

Comment:
it looks like the `s state.State` is unused.  Should this arg be removed or marked in some way to show it's not used?

Reply:
good point - fixed